syntax = "proto2";
package phxbinlog;

message PaxosEventData
{
	required string current_gtid = 1;
	optional string event_gtid = 2;
	required uint32 svrid = 3;
	optional bytes buffer = 4;
	optional uint64 checksum = 5;
};

message PaxosNodeInfo
{
	required string ip = 1;
	required uint32 port = 2;
};
